demisto_sdk/commands/validate/validators/PB_validators/PB100_is_no_rolename.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsNoRolenameValidator(BaseValidator[ContentTypes]):
+    error_code = "PB100"
+    description = "Validate whether the playbook has a rolename"
+    error_message = "The playbook '{playbook_name}' can not have a rolename."
+    related_field = "rolename"
+    is_auto_fixable = False
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        """Check whether the playbook has a rolename. If the Playbook has a rolename it is not valid.
+        Args:
+            - content_items (Iterable[ContentTypes]): The content items to check.
+        Return:
+            - List[ValidationResult]. List of ValidationResults objects.
+        """
```

Comment:
no need for doc strings for the is_valid / fix functions since we have the description field and they always receive/return the same thing.